pages/packingDetailed.js

```diff
@@ -0,0 +1,333 @@
+      <div className="bg-green-100 border border-green-400 text-green-700 px-4 py-3 rounded relative mb-3">
+        {this.state.success}
+      </div>
+    );
+
+    return (
+      <>
+        <Layout>
+          <div className="flex h-full">
+            <div className="w-1/4 bg-gray-200 items-center p-5">
+              <h1 className="text-xl">Pantry Note</h1>
+              <textarea
+                class="form-control w-full text-base font-normal text-gray-700 bg-white bg-clip-padding border border-solid border-gray-300 rounded transition ease-in-out m-0 focus:text-gray-700 focus:bg-white focus:border-blue-600 focus:outline-none"
+                id="pantry_note"
+                rows="4"
+                placeholder="Leave a note here for other pantry workers!"
+              >
+                {this.state.pantryNote}
+              </textarea>
+              <div>
+                <React.Fragment>
+                  <button
+                    class="bg-blue-500 hover:bg-blue-700 text-white py-1 px-2 border border-blue-700 rounded"
+                    onClick={() =>
+                      this.savePantryNote(
+                        document.getElementById("pantry_note").value
+                      )
+                    }
+                  >
+                    Save
+                  </button>
+                  <button
+                    class="bg-white-500 hover:bg-white-700 text-blue py-1 px-2 border border-blue-700 rounded"
+                    onClick={() => this.cancelPantryNote()}
+                  >
+                    Cancel
+                  </button>
+                </React.Fragment>
+              </div>
+            </div>
+            <div className="w-3/4 p-5 space-x-10 space-y-5">
+              {this.state.error ? errorBanner : null}
+              {this.state.success ? successBanner : null}
+              <h1 className="inline text-2xl font-medium mb-2">
+                {this.state.firstName + " " + this.state.lastInitial}{" "}
+              </h1>
+              <div className="inline items-center space-x-4">
+                <h2 className="inline text-l font-medium mb-2">
+                  {this.state.delivery_date}{" "}
+                </h2>
+                {this.displayOrderStatus()}
+                {this.displayChangeOrderStatus()}
+              </div>
+              <div className="text-large font-medium">
+                {"Additional Note: " + this.state.guestNote}
+              </div>
+              <table className="w-full table-fixed my-5" id="order">
+                <thead>
+                  <tr className="border-b-2">
+                    <th className="w-auto text-left">Item</th>
+                    <th className="w-1/6 text-left">Quantity</th>
+                    <th className="w-5"></th>
+                  </tr>
+                </thead>
+                <tbody className="divide-y">
+                  {Object.entries(this.state.items).map(([barcode, value]) =>
```

Comment:
I don't see why we'd ever have orders with no items in it, but just in case of some sort of weird stuff... can we make it so that it doesn't crash if we try to open an order with no items? i.e. when opening the jake order that's in firebase right now

Reply:
Just added some code for this! 
<img width="1677" alt="Screen Shot 2022-04-08 at 12 02 50 AM" src="https://user-images.githubusercontent.com/36006871/162382275-417d59a2-c583-40e5-9b8e-4f60f60678b5.png">